app/forms/users/invitation/principal_step/form.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+module Users::Invitation::PrincipalStep
+  class Form < ApplicationForm
+    include OpenProject::StaticRouting::UrlHelpers
+    include Redmine::I18n
+
+    form do |f|
+      f.hidden name: :project_id
+      f.hidden name: :principal_type
+
+      f.autocompleter(
+        name: :id_or_email,
+        label: name_label,
+        required: true,
+        autocomplete_options: {
+          defaultData: true,
+          component: "opce-members-autocompleter",
+          url: autocomplete_for_member_project_members_path(model.project_id, format: :json, type: model.principal_type),
+          focusDirectly: false,
+          multiple: false,
+          clearable: false,
+          appendTo: "##{Users::Invitation::DialogComponent::DIALOG_ID}"
+        }
+      )
+
+      f.autocompleter(
+        name: :role_id,
+        required: true,
+        include_blank: false,
+        input_width: :meidum,
```

Comment:
This is still a typo.
```suggestion
        input_width: :medium,
```

Reply:
Did you look at it commit by commit? This diff is already outdated

Reply:
Mhh.. don't know.. I thought, I was on the latest commit, but obviously not 🤷‍♂️